M2/Macaulay2/packages/GeometricDecomposability.m2

```diff
@@ -4,7 +4,7 @@ newPackage(
         "GeometricDecomposability",
         Version => "1.4.1",
         Date => "May 7, 2024",
-        Headline => "A package to check whether ideals are geometrically vertex decomposable",
+        Headline => "checking whether ideals are geometrically vertex decomposable",
```

Comment:
I'm trying to make all package headlines uniform, which includes removing redundant clauses. @mgacummings is this change okay with you?

Reply:
Sounds good! No issues here.